Code/GraphMol/MolEnumerator/Wrap/rdMolEnumerator.cpp

```diff
@@ -98,4 +117,54 @@ BOOST_PYTHON_MODULE(rdMolEnumerator) {
 Limitations:
   - Overlapping SRUs, i.e. where one monomer is contained within another, are
     not supported)DOC");
+
+  python::class_<MolEnumerator::StereoEnumerationOptions>(
+      "StereoEnumerationOptionsV2", "Stereoisomer enumeration parameters",
```

Comment:
Why V2?

Reply:
Wanted to allow for testing without deleting the previous implementation. I'd like to figure out how to handle the random isomer functionality as part of this review